.github/workflows/hackage.yml

```diff
@@ -0,0 +1,127 @@
+name: Hackage
+
+on:
+  push:
+    branches:
+      - '*-hackage'
+
+jobs:
+  check-and-upload-tarballs:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package: ["hie-compat", "hls-graph", "shake-bench",
```

Comment:
Looks like this list needs to be kept up to date when new plugins are added? Do we have any documentation on how to add new plugins? If so, we should document this fact there too.

Reply:
Yeah, fortunately plugins have an unique id and it is relatively easy search in the codebase using it. So search for a similar existing plugin is a required step in front of outdated documentation. But you are right, i guess the best place to list all the places (including .cabal and generic config) could be https://haskell-language-server.readthedocs.io/en/latest/contributing/plugin-tutorial.html#wrapping-up

Reply:
@isovector #2166